lit/index.lit

```diff
@@ -123,16 +108,16 @@
     }}}
   }
 
-  \splash-example{Rapid local iteration}{
-    The \reference{fly-execute} command lets you run a build with local changes.
+  \splash-example{Efficient local testing}{
+    The \reference{fly-execute} command enables you to run builds with local modifications.
 
-    This build runs in exactly the same way as it would run in your pipeline,
-    without having to push broken commits until it works.
+    These builds execute exactly as they would in your pipeline, eliminating the
+    need to push incomplete commits while debugging.
 
-    When a build in the pipeline fails, you can run \reference{fly-execute} with
-    the \code{-j} flag to run a one-off build with the same inputs as the failed
-    build. You can then replace an input with your local changes with \code{-i}
-    to see if your fix is valid.
+    When a pipeline build fails, you can use \reference{fly-execute} with
+    the \code{-j} flag to run a one-off build with identical inputs as the failed
```

Comment:
```suggestion
    the \code{-j} flag to run a one-off build with the same inputs as the failed
```

"identical" implies there's a difference between the inputs used in the build and what we're using with `fly execute` when that's not true. Original phrasing was fine imo.